config/sync/core.extension.yml

```diff
@@ -133,7 +133,6 @@ module:
   oembed_providers: 0
   options: 0
   page_cache: 0
-  page_cache_query_ignore: 0
```

Comment:
Gotta delete the file I told you about.

Reply:
Yeah, sorry. I jumped the gun on the review request. It's been a morning.